packages/core/src/components/Card/Header/Header.tsx

```diff
@@ -27,17 +27,17 @@ export interface HvCardHeaderProps
   classes?: HvCardHeaderClasses;
 }
 
-export const HvCardHeader = ({
-  classes: classesProp,
-  className,
-  title,
-  subheader,
-  icon,
-  onClick,
-  ...others
-}: HvCardHeaderProps) => {
-  const { activeTheme } = useTheme();
-  const { classes, css, cx } = useClasses(classesProp);
+export const HvCardHeader = (props: HvCardHeaderProps) => {
+  const {
+    classes: classesProp,
+    className,
+    title,
+    subheader,
+    icon,
+    onClick,
+    ...others
+  } = useDefaultProps("HvCardHeader", props);
```

Comment:
I think we should include these improvements & missing `useDefaultProps` in a separate PR, to reduce the scope & changes of this huge PR

Reply:
In this case it was needed to remove on `titleVariant` theme variable

Reply:
I think we should definitely expose `HvCardHeader` - it's part of our API. I missed it 😞